lib/helper.js

```diff
@@ -0,0 +1,12 @@
+var semver = require('semver');
+
+exports.DEFAULT_LB_VERSION = '2.x';
+
+/**
+ * validate loopback version
+ * @param version
+ * @returns valid loopback version or default if invalid
+ */
+exports.validateLoopbackVersion = function(version) {
```

Comment:
Actually, since this is converting an npm dependency string into a `2.x`/`3.x` value, I think a name starting with `validate` is misleading. How about `parseLoopBackVersion` or `parseLoopBackDependency` or similar?